Detectors/TPC/base/include/TPCBase/Sector.h

```diff
@@ -26,17 +26,15 @@ namespace TPC {
 class Sector
 {
   public:
-    enum
-    {
-        MaxSector = 36
-    };
+    // the number of sectors
+    static constexpr char MAXSECTOR=36;
```

Comment:
I always think of characters when I read `char`, so I would use a different data type for readability. Just a matter of taste, internally its the same.